### PR TITLE
Add missing x-ms-parameter-location

### PR DESCRIPTION
**Pull request by jeremymeng**
for `PathRenameMode` in blob.json and `FileRangeWriteFromUrl` in file.json

### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/storage/data-plane/Microsoft.FileStorage/preview/2019-02-02/file.json [hunk 1]
```diff
@@ -4214,6 +4214,7 @@
             "description": "Only update is supported: - Update: Writes the bytes downloaded from the source url into the specified range.",
             "required": true,
             "type": "string",
+            "x-ms-parameter-location": "method"
             "enum": [
                 "update"
             ],
```
Comment (tg-msft) [inline specification/storage/data-plane/Microsoft.FileStorage/preview/2019-02-02/file.json]: This is invalid json, right?  Don't you need a comma after the location and is the diff saying you deleted the colon?

Comment (jeremymeng) [reply specification/storage/data-plane/Microsoft.FileStorage/preview/2019-02-02/file.json]: Aha, I was wondering what the red spot trying to tell me!  Fixed.

Comment (tg-msft) [reply specification/storage/data-plane/Microsoft.FileStorage/preview/2019-02-02/file.json]: Ohhhh, that's what the hell the weird red colon was?  I couldn't figure that out either.

**Review by tg-msft — APPROVED** [review]

**Comment by akning-ms** [comment]
Hi @jeremymeng, Can we know what is last status for this PR as it is open for a long time? Do you have issue to fix validation error?

**Comment by jeremymeng** [comment]
I am going to close this for now.  We worked around this issue using autorest transform.  There's already a newer version (2019-07-07) swagger json files for Blob and File Share. This fix should be made against those.

/cc @XiaoningLiu @jiacfan

**Pull request closed without merging**